lib/language_pack/rails6.rb

```diff
@@ -14,6 +14,32 @@ def self.use?
     end
   end
 
+  def node_modules_folder
+    "node_modules"
+  end
+
+  def public_packs_folder
+    "public/packs"
+  end
+
+  def webpacker_cache_folder
+    "tmp/cache/webpacker"
```

Comment:
This path is the [default value](https://github.com/rails/webpacker/blob/f9c29dd08268de6bddf4de50422258864938f75c/test/mounted_app/test/dummy/config/webpacker.yml#L7) for where Webpacker caches assets. As the other paths are hardcoded, I think this is acceptable.

Reply:
```rb
Webpacker.config.cache_path
```

^ the above should give you the cache path.

https://github.com/rails/webpacker/blob/6efb3ee5367dbced7b76b7daec35684db4d9570a/lib/webpacker/configuration.rb#L50

Reply:
@ParamagicDev This is a bit tricker then I first thought 😬 

To check if webpacker is installed & get that info, I came up with:

```
bundle exec rails runner 'puts Webpacker.config.cache_path if defined?(Webpacker)'  
```

Though, I'm not a fan as if webpacker was to change that method we'd have to update this code. I think it's more resilient to just hardcode the path & let people who stick to defaults feel the speed.